### ISSUE TITLE
Allow to toggle full commit diff from within the diff view

### ISSUE THREAD

**Issue by axelson** [jonas/tig#356]
Currently if you open up a path such as `tig ./src` then you see all commits that modified files in that path. So far so good. But if you then view one of those commits by pressing `enter` you see the changes in that commit that modify the path but not the changes in the commit that modify files outside of the path.

Is there any way to switch to the full commit view once you are looking at the partial commit?

Also, thanks for Tig! It's great.
I tried to search for this issue but wasn't able to find anything relevant.

**Comment by jonas** [comment]
Yes, having this information would be useful and if it is in the diff view it would even be toggled with Enter.

**Comment by JDarnley** [comment]
I just found this comment so I don't need to open my own issue for the very same topic.  Thanks for pointing out % it will be extremely useful.
